cmgd/cmgd_vty.c

```diff
@@ -207,6 +207,40 @@ DEFPY(cmgd_commit,
 	return CMD_SUCCESS;
 }
 
+static cmgd_database_id_t get_database(const char *db_name)
```

Comment:
Can you move this to cmd_db.c/.h as
cmgd_database_id_t cmgd_get_database_by_name(const char *db_name);